libs/core/src/scss/base/_variables.scss

```diff
@@ -5,6 +5,7 @@
   SPACINGS
 ****************************************************************************/
 $sizes: (
+  xxxxl: 64px,
```

Comment:
Is this size aligned with UX, so it is available in Sketch/Zeplin too, for designers to use?

Reply:
The addition was on behalf of @henrikvoetmand - will ask him your question...